text/chapter5.md

```diff
@@ -206,20 +184,32 @@ Note that we could have also written
 
 and PSCi would have inferred the same type.
 
+## Record Puns
+
+Recall that the `showPerson` function matches a record inside its argument, binding the `first` and `last` fields to values named `x` and `y`. We could alternatively just reuse the field names themselves, and simplify this sort of pattern match as follows:
```

Comment:
New home for this section. Also converted from `Point` to `Person` examples to better match the previous section.
I was considering converting all record examples from `Person` to `Point` to maintain the "shapes" theme of the chapter, but that would involve lots of edits to the previous section too.